source/solutions-library/pdf-search.txt

```diff
@@ -186,53 +180,51 @@ Superduper.io
 framework for integrating AI models and workflows directly with and
 across major databases for more flexible and scalable custom enterprise
 AI solutions. It enables developers to build, deploy, and manage AI on
-their existing data infrastructure and data, while using their preferred
+their existing infrastructure and data while using their preferred
 tools, eliminating data migration and duplication.
 
 **With Superduper.io, developers can:**
 
-- Bring AI to their databases, eliminating data pipelines and moving
-  data, minimizing engineering efforts, time to production, and
+- Bring AI to their databases, which eliminates data pipelines and
+  minimizes engineering efforts, time to production, and
   computation resources.
 
-- Implement AI workflows with any AI models and APIs, on any type of
-  data, with any AI and Python framework, package, class, or function.
+- Implement AI workflows with any AI models and APIs, and on any type of
+  data.
 
-- Safeguard data by switching from APIs to hosting and fine-tuning your
-  own models, on your own and existing infrastructure, whether
-  on-premises or in the cloud.
+- Safeguard data by switching from APIs to hosting and fine-tuning their
+  own models on their own infrastructure.
 
-- Easily switch between embedding models and LLMs to other API providers
-  as well as hosting your own models on HuggingFace or elsewhere just by
-  changing a small configuration.
+- Easily switch from embedding models to other API providers. They can
+  also host their models on HuggingFace or other platforms.
 
 Superduper.io provides an array of sample use cases and notebooks that
 developers can use to get started, including vector search with MongoDB,
 embedding generation, multimodal search, RAG, transfer learning, and
-many more. The demo showcased in this solution is adapted from an `app
-previously developed <https://github.com/SuperDuperDB/poc-volvo>`__ by
+many more. This solution's demo is adapted from a
+`previously developed app <https://github.com/SuperDuperDB/poc-volvo>`__ from
 Superduper.io. 
 
 Key Learnings
 -------------
 
+.. MAYA: this should go somewhere before key learnings.
+
 - Build the solution following the instructions in this `Github repo
   <https://github.com/mongodb-industry-solutions/Insurance-PDF-Search/tree/main>`__.
-  It is important to note that the solution is made of two logical
-  steps:
+  The solution is composed of two steps:
 
   - The initialization script breaks down the PDFs into chunks and then
     turns them into vector embeddings.
   - The querying step allows the user to interrogate the documents.
```

Comment:
Optional but maybe this should be an ordered list, if this is a list of steps (even if it's only 2)